dsl/camel-jbang/camel-jbang-core/src/main/java/org/apache/camel/dsl/jbang/core/common/RuntimeUtil.java

```diff
@@ -24,10 +24,7 @@
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
-import java.util.StringJoiner;
+import java.util.*;
```

Comment:
This one might cause the style check to go go crazy.

Reply:
BTW, no need to hold the PRs for this. This is just a suggestion and not a big deal.